vllm/v1/engine/detokenizer.py

```diff
@@ -28,8 +30,9 @@
 
 class IncrementalDetokenizer:
 
-    def __init__(self):
+    def __init__(self, vllm_config: VllmConfig):
         self.token_ids: list[int] = []
+        self.vllm_config = vllm_config
```

Comment:
I think it could just go in `BaseIncrementalDetokenizer`, it's not needed here right?